solutions/documentation.md

```diff
@@ -0,0 +1,55 @@
+# Recommended solutions for Documentation
+
+Patterns for serving documentation pages to our communities and clients.
+
+## Community documentation
+
+The majority of our documentation will be community-focused. This means both that the various communities around our products are using the documentation, and that we hope they will also help to improve the documentation.
+
+### Discourse forums for content
+
+We recommend that community documentation content should live in [Discourse forums](https://www.discourse.org/). Discourse forums provide a useful model for building a community around our content.
+
+This content could live inside a category inside [discourse.ubuntu.com](https://discourse.ubuntu.com/) or any other Discourse installation.
+
+#### Structure
+
+Documentation content should be stored in ["wiki" posts](https://meta.discourse.org/t/what-is-a-wiki-post/30801) that users can comment on, and when they build up enough trust, can edit directly. The [trust levels](https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/) have configurable thresholds. Documentation topics should be grouped under an appropriately-named [category](https://meta.discourse.org/t/how-to-add-categories/71859) - e.g. `docs` or `product-docs`.
+
+Each documentation set shuold have an introductory page. At the bottom of this page should be a horizontal rule, followed by the heading "Content", and then a navigation list which should link to other documentation pages in the forum, e.g.:
+
+``` markdown
+Hello. Welcome to our documentation.
+
+Please spend a while here.
+
+---
+
+# Content
+
+- [Getting started](/t/getting-started/3876)
+- [Configuration options](/t/configuration-options/87)
+
+```
+
+#### Branching documentation
+
+Every effort should be made to keep one canonical set of documentation. When different versions of software need to be support, any differences should be mentioned inline in the documents.
+
+If branches of documentation must be created, e.g. when a new major version of software, additional categories can be created for the old version - e.g. `docs-v1.x` alongside `docs`. The title of this older content should contain the version number - e.g. `v1.x: Getting started`.
+
+#### Plugins
+
+We discourage the use of plugins in Discourse unless absolutely necessary, as it's easier to manage multiple Discourse installations if they remain as similar as possible.
+
+### The website application
+
+Documentation content will be pulled from the Discourse forum to be displayed on our official documentation websites - e.g. at https://maas.io/docs - which will be written and managed by the Web and Design team.
```

Comment:
Instead of showing an example to maas docs. Maybe link to docs.ubuntu.com would be a better canonical link to official documentation websites.